src/prisma/prisma.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client';
+
+@Injectable()
+export class PrismaService extends PrismaClient implements OnModuleInit {
```

Comment:
**praise**: Extending `PrismaClient` and implementing `OnModuleInit` is a clean way to manage database connections.